src/API/Authentication.php

```diff
@@ -52,6 +52,13 @@ class Authentication
      */
     private $audience;
 
+    /**
+     * Organization ID, if applicable.
```

Comment:
Let's add a _tiny_ bit more context here. For example, elsewhere we've used:

> The Id of an organization to log in to

Reply:
Good call, improved that with your example. Also expanded on src/Auth0.php's explenaton a bit as well.